models/user.schema.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+  userId: String,
```

Comment:
Use the reserved _id field instead of creating a userId field

Reply:
We removed the userId. For _id that's automatically added as a property without us having to explicitly write it correct?